apps/aecore/test/aecore_txs_SUITE.erl

```diff
@@ -72,54 +72,73 @@ txs_gc(Config) ->
 
     %% Mine a block to get some funds. Height=1
     aecore_suite_utils:mine_key_blocks(N1, 1),
+    Height0 = 1,
 
     %% Add a bunch of transactions...
-    add_spend_tx(N1, 1000, 1,  1,  10), %% Ok
-    add_spend_tx(N1, 1000, 1,  2,  10), %% Should expire ?EXPIRE_TX_TTL after
-                                        %% next TX is on chain = 2 + 2 = 4
-    add_spend_tx(N1, 1000, 10, 2,  10), %% Duplicate should be preferred
-    add_spend_tx(N1, 1000, 1,  3,  10), %% Ok
+    {ok, TxH1} = add_spend_tx(N1, 1000, 1,  1,  10), %% Ok
+    {ok, _}    = add_spend_tx(N1, 1000, 1,  2,  10), %% Should expire ?EXPIRE_TX_TTL after
+                                                     %% next TX is on chain = 2 + 2 = 4
+    {ok, TxH2} = add_spend_tx(N1, 1000, 10, 2,  10), %% Duplicate should be preferred
+    {ok, TxH3} = add_spend_tx(N1, 1000, 1,  3,  10), %% Ok
 
-    add_spend_tx(N1, 1000, 1,  5,  10), %% Non consecutive nonce
-    add_spend_tx(N1, 1000, 1,  7,  10), %% Non consecutive nonce
-    add_spend_tx(N1, 1000, 1,  8,  5),  %% Short TTL - expires at 5 + 2 = 7
+    {ok, TxH5} = add_spend_tx(N1, 1000, 1,  5,  10), %% Non consecutive nonce
+    {ok, _}    = add_spend_tx(N1, 1000, 1,  7,  10), %% Non consecutive nonce
+    {ok, _}    = add_spend_tx(N1, 1000, 1,  8,  5),  %% Short TTL - expires at 5 + 2 = 7
 
     %% Now there should be 7 transactions in mempool
     {ok, Txs1} = rpc:call(N1, aec_tx_pool, peek, [infinity]),
     {7, _} = {length(Txs1), Txs1},
 
-    %% Mine two key blocks to get a micro-block in between. Height=3
-    aecore_suite_utils:mine_key_blocks(N1, 2),
+    %% Mine to get TxH1-3 onto chain
+    {ok, Blocks1} = aecore_suite_utils:mine_blocks_until_txs_on_chain(N1, [TxH1, TxH2, TxH3], 10),
```

Comment:
This only counts key blocks. Ok.

Reply:
Yes, `mine_blocks_until_tx(s)_on_chain` only returns key-blocks.